projects/assets/src/translations/en/cart.ts

```diff
@@ -25,6 +25,7 @@ export const cart = {
     salesTax: 'Sales Tax:',
     grossTax: 'The order total does not include tax of',
     total: 'Total:',
+    TBD: 'To be determined',
```

Comment:
put it TBD as how it was hardcoded

but this allows for flexible as user can change the value of this key now

Reply:
Also, assign it to me in the issues instead of the PR

Once I comment on it or even approve, it will put me as reviewer

Reply:
Changes have been made.

Reply:
Sorry my mistake :) 

Can you change the key to toBeDetermined. Sorry kind of took a quick peek last time on the string only!

Reply:
Corrected.